src/utils/utils-vega.js

```diff
@@ -160,6 +160,10 @@ export function createVegaAttrMixin(
         } else {
           quantScale.domain(domainVals).range(capAttrObj.range)
           rtnVal = quantScale(input)
+
+          if (typeof rtnVal === "undefined") {
+            rtnVal = layerObj[defaultFunc]()
```

Comment:
@bmatcuk, I don't think we can use default size here. This bug is caused by setting the SIZE DOMAIN min/max value to the same value which caused d3 quantize scaling got infinity value. There is SIZE RANGE slider just underneath the SIZE DOMAIN which is tightly dependent one another. Assuming user changed the SIZE RANGE to some other value other than default min 3, then setting the SIZE DOMAIN to min=1 max=1 as in the issue, then your fix with default would be mismatch. I think we can use different scale, but @biovisualize  is our d3 expert.